gcc/rust/parse/rust-parse-impl.h

```diff
@@ -11637,6 +11637,11 @@ Parser<ManagedTokenSource>::parse_stmt_or_expr ()
 	AST::PathInExpression path = parse_path_in_expression ();
 	std::unique_ptr<AST::Expr> null_denotation;
 
+	if (path.is_error ())
+	  {
+	    rust_error_at (path.get_locus (), "expected identifier");
+	  }
```

Comment:
```suggestion
	    rust_error_at (path.get_locus (), "expected identifier");
```

small formatting nit :)